src/commands/dev/generate/flag.ts

```diff
@@ -0,0 +1,342 @@
+          if (!input) return true;
+          return Number.isInteger(Number(input)) ? true : 'Must be an integer';
+        },
+      },
+      {
+        type: 'list',
+        name: 'salesforceIdLength',
+        message: 'Required length for salesforceId',
+        choices: ['15', '18', 'None'],
+        when: (ans: Answers): boolean => ans.type === 'salesforceId',
+      },
+      {
+        type: 'input',
+        name: 'salesforceIdStartsWith',
+        message: 'Required 3 character prefix for salesforceId (optional)',
+        when: (ans: Answers): boolean => ans.type === 'salesforceId',
+        validate: (input: string): string | boolean => {
+          if (!input) return true;
+          return input.length === 3 ? true : 'Must be 3 characters';
+        },
+      },
+      {
+        type: 'confirm',
+        name: 'fileOrDirExists',
+        message: 'Does this flag require the file or directory to exist',
+        default: false,
+        when: (ans: Answers): boolean => ['file', 'directory'].includes(ans.type),
+      },
+      {
+        type: 'input',
+        name: 'integerMin',
+        message: 'Minimum required value for integer flag (optional)',
+        when: (ans: Answers): boolean => ans.type === 'integer',
+        validate: (input: string): string | boolean => {
+          if (!input) return true;
+          return Number.isInteger(Number(input)) ? true : 'Must be an integer';
+        },
+      },
+      {
+        type: 'input',
+        name: 'integerMax',
+        message: 'Maximum required value for integer flag (optional)',
+        when: (ans: Answers): boolean => ans.type === 'integer',
+        validate: (input: string): string | boolean => {
+          if (!input) return true;
+          return Number.isInteger(Number(input)) ? true : 'Must be an integer';
+        },
+      },
+    ]);
+  }
+
+  private async loadExistingFlags(commandId: string): Promise<Record<string, Interfaces.Command.Flag>> {
+    const config = new Config({ root: process.cwd() });
+    config.root = config.options.root;
+    await config.load();
+    const cmd = config.commands.find((c) => c.id === commandId);
+    return cmd.flags ?? {};
+  }
+
+  private async findExistingCommands(): Promise<string[]> {
+    return (await fg('src/commands/**/*.ts'))
+      .map((file) => {
+        const p = path.parse(file.replace(path.join('.', 'src', 'commands'), ''));
+        const topics = p.dir.split('/');
+        const command = p.name !== 'index' && p.name;
+        const id = [...topics, command].filter((f) => f).join(':');
+        return id === '' ? '.' : id;
+      })
+      .sort()
+      .map((c) => toConfiguredId(c, this.config));
+  }
+
+  private async updateMarkdownFile(commandName: string, flagName: string): Promise<void> {
+    const filePath = path.join('messages', `${commandName.split(':').join('.')}.md`);
+    await fs.appendFile(filePath, `\n# flags.${flagName}.summary\n\nDescription of ${flagName}.\n`);
```

Comment:
We could prompt for the description and insert it for them. That way we could enforce message standards with inquirer: 
- starts with a capital
- ends with a period
- less than X characters long (75?)

Reply:
@iowillhoit I thought about that.

I was concerned that asking that would slow down development too much. As a developer, I might not want to take the time to think through a meaningful description for a flag while in the middle of building my command

What do you think?

Reply:
Yea I can see that side of it too. Since we are holding off on the markdown linting for now, I think it would be a nice way to enforce some standards. What the description prompt was: 

`Flag description (press return for placeholder)` 

That way they could skip it if they wanted? Either way, I am fine with whatever you decide Mike